### PR TITLE
Body

### PR DESCRIPTION
**Pull request by Thoms-C**
This is the body for the first pillar (Why FAVART?) of the project

### CODE REVIEW

**Review by Marie-Garde — DISMISSED** [review]
Nickel

**Review by Marie-Garde — APPROVED** [review]
ok

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...ngCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\index.js",[],["27","28"],"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\reportWebVitals.js",[],"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\App.js",[],"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\components\\Main.jsx",[],"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\Styled-components\\MainStyle.js",[],"C:\\Users\\thoma\\desktop\\projet-favart\\lille-0920-js-p3-favart\\src\\Assets\\Device\\Device.jsx",[],{"ruleId":"29","replacedBy":"30"},{"ruleId":"31","replacedBy":"32"},"no-native-reassign",["33"],"no-negated-in-lhs",["34"],"no-global-assign","no-unsafe-negation"]
```
Comment (LoicBrassart) [inline .eslintcache:1]: I advise you to add this file to your `.gitignore`: I now know your name, your OS, that you don't tidy up your desktop ( :p ), ...
_Edit: I stand corrected, you did just that. I'll search for a way to remove this file from the repo_

File: src/Assets/Device/Device.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const size = {
+  smallDevices: "576px",
+  mediumDevices: "768px",
+  largeDevices: "992px",
+  extraLargeDevices: "1200px",
```
Comment (LoicBrassart) [inline src/Assets/Device/Device.jsx:5]: (Hard-coded values could have been added to the .env file)

**Review by LoicBrassart — APPROVED** [review]
No Real Evil

**Pull request merged**